Extension/src/LanguageServer/client.ts

```diff
@@ -807,6 +815,83 @@ export class DefaultClient implements Client {
         };
     }
 
+    async promptSelectCompiler(command: boolean | undefined): Promise<void> {
+        const selectCompiler: string = localize("selectCompiler.string", "Select Compiler");
+        const confirmCompiler: string = localize("selectCompiler.string", "Yes");
```

Comment:
Every call to localize must use a unique ID (the first arg).  Otherwise, only one of these strings will get localized.